docs/managed_vms/endpoints/README.md

```diff
@@ -1,8 +1,27 @@
 # Google Cloud Endpoints sample
 
+WARNING: This sample does not work with Go 1.7 yet. Please use Go 1.6.
```

Comment:
There is an issue with aedeploy and Go 1.7. I filed a bug here: https://code.google.com/p/googleappengine/issues/detail?id=13222